server/src/services/JobService.ts

```diff
@@ -0,0 +1,48 @@
+import { Context } from "../context";
+import { AuthUtil } from "../util/AuthUtil";
+
+interface PostJob {
+  title: string;
+  description: string;
+  price: number;
+}
+
+export class JobService {
+  public static async getJobs(ctx: Context) {
+    try {
+      const user = await AuthUtil.verifyAndGetUser(ctx);
+
+      if (!user) {
+        throw new Error("Error retrieving logged in user");
+      }
```

Comment:
this is not required because if user does not exist, the function verifyAndGetUser will throw an error, in this case you dont even have to set `await AuthUtil.verifyAndGetUser(ctx)` to a variable, just run the function and it will validate the session.